pkg/sql/schema_changer_test.go

```diff
@@ -2981,6 +2981,39 @@ func TestAddComputedColumn(t *testing.T) {
 	sqlDB.CheckQueryResults(t, `SELECT * FROM t.test ORDER BY a`, [][]string{{"2", "7"}, {"10", "15"}})
 }
 
+// TestDropComputedColumn verifies the effects of removing the
+// computed-ness from a column.
+func TestDropComputedColumn(t *testing.T) {
```

Comment:
this should be a logic test instead. i don't think there's anything here we can't do there.

Reply:
Can you point me at an example of what this would look like?  I'm not immediately sure how one would go about testing this without going through the SQL layer.